### PR TITLE
feat(docs): FRMW-173 - Migrate rxEnvironment docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-173

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

**Comment by 100stacks** [comment]
LGTM

File: src/rxEnvironment/rxEnvironment.js [hunk 1]
```diff
@@ -4,7 +4,92 @@
...
+ *
+ * ## Current Environments ##
+ *
+ * This service defines the following Encore specific environments:
+ *
+ * * **`local`** - http://localhost:port and http://server:port
+ * * **`preprod`** - http://preprod.encore.rackspace.com
+ * * **`unified-preprod`** - https://*.encore.rackspace.com
+ * * **`unified`** - All environments including https://encore.rackspace.com
+ * * **`unified-prod`** - Only https://encore.rackspace.com
```
Comment (CITguy) [inline src/rxEnvironment/rxEnvironment.js]: Please remove backticks for bold list items above.

```
* **local** - http://localhost:port and http://server:port
* **preprod** - http://preprod.encore.rackspace.com
...
```

File: src/rxEnvironment/rxEnvironment.js [hunk 2]
```diff
@@ -4,7 +4,92 @@
...
+ * and take special actions if `Environment.isPreProd()` or
+ * `Environment.isUnifiedProd()` are `true`.
+ *
+ * ## Overlapping Environments ##
+ *
+ * Keep in mind that the environments we define are not mutually exclusive. For
+ * instance, if we're at `http://preprod.encore.rackspace.com`, then we are in
+ * the `preprod` environment, the `unified-preprod` environment, and `unified-prod`.
+ *
+ * ## Adding New Environments ##
```
Comment (CITguy) [inline src/rxEnvironment/rxEnvironment.js]: I feel like this section ("Adding New Environments") belongs in the documentation for the `Environment` service.

**Comment by bry4n** [comment]
@CITguy  updated

**Comment by CITguy** [comment]
LGTM

**Pull request merged**